docs/app/Components/ComponentDoc/ComponentExample.js

```diff
@@ -129,7 +129,7 @@ class ComponentExample extends Component {
   }
 
   copyJSX = () => {
-    copyToClipboard(this.getOriginalSourceCode())
+    copyToClipboard(this.state.sourceCode)
```

Comment:
Bug fix, an example's copy button was copying the original source code.  This update makes it copy the current state of the editor.